packages/ui-app/src/styles/app.css

```diff
@@ -108,7 +108,7 @@ article {
   }
 
   &:not(:hover) {
-    .ui.button {
+    .ui.button:not(.disabled) {
```

Comment:
Think it is fine without? It is just to indicate the presence, not the state. Or opacity giving issues?

Reply:
Yes it was opacity )